app/models/constants/case_log.rb

```diff
@@ -198,23 +203,22 @@ module Constants::CaseLog
   }.freeze
 
   BENEFITS = {
-    "All" => 1,
-    "Some" => 2,
-    "None" => 3,
-    "Don’t know" => 4,
+    "1. All" => 1,
```

Comment:
@paulrobertlloyd / @roobottom can confirm but I think the decision was not to include the numbers in answer options until a further spike has been done on the paper form?

Reply:
Correct. 🤞we never have to include the numbers on the service.

Reply:
Yesterday afternoon! Removed from the prototype, still need to update the map (that might be a while!)

Reply:
😭 Alright, I'll get rid of them. I take it the wording still stays the same?

Reply:
Yup. Just not showing the numbers at the beginning of each question.

Reply:
Sweet, thank you thank you!